src/components/organize/tasks/forms/TaskDetailsForm.tsx

```diff
@@ -46,6 +47,22 @@ const TaskDetailsForm = ({ onSubmit, onCancel, task }: TaskDetailsFormProps): JS
         if (!values.campaign_id) {
             errors.campaign_id = intl.formatMessage({ id: 'misc.formDialog.required' });
         }
+        if (values.type === TASK_TYPE.VISIT_LINK || values.type === TASK_TYPE.SHARE_LINK) {
+            const config = values.config as VisitLinkConfig;
+            if (config?.url) {
+                const valid = validator.isURL(config.url);
```

Comment:
There is a bug here, because by default the `validator.isURL()` function considers URLs without protocol ( e.g. `google.com`) valid. But that will not work down the line in the rest of the system, so the URL must be verified to contain protocol (e.g. `https://google.com`).

Configure `validator.isURL()` to verify that protocol is included.

Reply:
Added the options 
`{
    protocols: ['http','https','ftp', 'ftps'],
    require_protocol: true,
}` 
to `isURL()`